City-of-Helsinki/drupal-helfi-platform-config
# UHF-6883 Kuura chat cookies

#  [UHF-6166](https://helsinkisolutionoffice.atlassian.net/browse/UHF-6166)
Kuura cookie settings and info.

## What was done
* Added kuura cookie info and settings.
* Added missing leijuke cookie info.

## How to install
* Make sure your instance is up and running on latest dev branch.
    * `git pull origin dev`
    * `make fresh`
* Update the Helfi Platform config
    * `composer require drupal/helfi_platform_config:dev-UHF-6166-kuura-chat-cookies`
* Run `make drush-updb drush-cr`

## How to test
* Check the cookie info /evasteasetukset page. Check the cookies are not deleted while running.
## Other PRs
*
